server/helpers/identification.js

```diff
@@ -0,0 +1,37 @@
+import { UserRole, User, Role } from '../database/models';
+
+const findIdByEmail = (email, response) => User.findOne({
+  where: {
+    email,
+  },
+  attributes: ['id']
```

Comment:
Why are you returning just the id?

Reply:
> Why are you returning just the id?

I am using the email to get the id of the account